cmd/substreams/run.go

```diff
@@ -29,6 +29,7 @@ func init() {
 	runCmd.Flags().StringSlice("debug-modules-initial-snapshot", nil, "List of 'store' modules from which to print the initial data snapshot (Unavailable in Production Mode")
 	runCmd.Flags().StringSlice("debug-modules-output", nil, "List of extra modules from which to print outputs, deltas and logs (Unavailable in Production Mode)")
 	runCmd.Flags().Bool("production-mode", false, "Enable Production Mode, with high-speed parallel processing")
+	runCmd.Flags().StringSliceP("params", "x", nil, "Set a parameters for supported modules. Can be used multiple times. Example: -x module1=valA -x module2=valX&valY")
```

Comment:
Here I propose we rather remove `-p` and `-k` and keep the long-form args for those flags, and we use `-p` here, for what needs to be more intuitive. The other ones are mostly for internal dev by the core team, in wonky environments.